.github/workflows/wasm.yml

```diff
@@ -0,0 +1,114 @@
+        run: |
+          set +e
+          emcc --version
+          ninja --version
+          gcc --version
+      - name: Checkout SerenityOS/libjs-test262-data libjs-wasm
+        uses: actions/checkout@v4
+        with:
+          repository: SerenityOS/libjs-test262-data
+          path: libjs-test262-data
+          ref: libjs-wasm
+          lfs: true
+      - name: "Create build directories"
+        run: |
+          mkdir -p ${{ github.workspace }}/Build/caches/TZDB
+          mkdir -p ${{ github.workspace }}/Build/caches/UCD
+          mkdir -p ${{ github.workspace }}/Build/caches/CLDR
+      - name: "TimeZoneData cache"
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/Build/caches/TZDB
+          key: TimeZoneData-${{ hashFiles('Meta/CMake/time_zone_data.cmake') }}
+      - name: "UnicodeData cache"
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/Build/caches/UCD
+          key: UnicodeData-${{ hashFiles('Meta/CMake/unicode_data.cmake') }}
+      - name: "UnicodeLocale cache"
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/Build/caches/CLDR
+          key: UnicodeLocale-${{ hashFiles('Meta/CMake/locale_data.cmake') }}
+      - name: "Build host lagom tools"
+        run: |
+          cmake -GNinja \
+            -B ${{ github.workspace }}/Build/lagom-tools \
+            -S ${{ github.workspace }}/Meta/Lagom \
+            -DBUILD_LAGOM=OFF \
+            -DSERENITY_CACHE_DIR=${{ github.workspace }}/Build/caches \
+            -DCMAKE_C_COMPILER=gcc-12 \
+            -DCMAKE_CXX_COMPILER=g++-12 \
+            -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/Build/lagom-tools \
+            -Dpackage=LagomTools
+
+          ninja -C ${{ github.workspace }}/Build/lagom-tools install
+      - name: "Create wasm build environment"
+        run: |
+          emcmake cmake -GNinja \
+            -B ${{ github.workspace }}/Build/wasm \
+            -S ${{ github.workspace }}/Meta/Lagom \
+            -DLagomTools_DIR=${{ github.workspace }}/Build/lagom-tools/share/LagomTools \
+            -DBUILD_LAGOM=ON \
+            -DSERENITY_CACHE_DIR=${{ github.workspace }}/Build/caches \
+            -DBUILD_SHARED_LIBS=OFF
+      - name: "Build libjs.{js,wasm}"
+        run: |
+          ninja -C ${{ github.workspace }}/Build/wasm libjs.js
+      - name: "Show me what you built :)"
+        run: |
+          ls -l ${{ github.workspace }}/Build/wasm
+
+      - name: "Prepare files"
+        run: |
+          cp ${{ github.workspace }}/Build/wasm/bin/libjs.js ${{ github.workspace }}/libjs-test262-data/libjs.js
+          cp ${{ github.workspace }}/Build/wasm/bin/libjs.wasm ${{ github.workspace }}/libjs-test262-data/libjs.wasm
+          echo 'Module.SERENITYOS_COMMIT = "${{ github.sha }}";' >> ${{ github.workspace }}/libjs-test262-data/libjs.js
+          tar -czvf libjs-wasm.tar.gz -C ${{ github.workspace }}/libjs-test262-data .
```

Comment:
This also adds `${{ github.workspace }}/libjs-test262-data/.git` to the published artifact.

Reply:
lol oops